src/api/common_api.js

```diff
@@ -252,41 +252,66 @@ module.exports = {
             }
         },
 
+        bucket_policy_principal: {
+            anyOf: [{
```

Comment:
I recall we didn't support anyOf/oneOf in postgresClient encode_json/decode_json, @dannyzaken is this valid now?

Reply:
@dannyzaken ?

Reply:
As we discussed, this anyOf doesn't really gets encoded, but since all we have inside the any of is string/sensitive string it works. 
@dannyzaken Are you ok with merging it and opening a gap for the anyOf issue?

Reply:
opened an issue for the anyOf: https://github.com/noobaa/noobaa-core/issues/7448

Reply:
I'm fine with the gap.